src/modal/test/modal.spec.js

```diff
@@ -142,6 +142,7 @@ describe('$modal', function () {
   function open(modalOptions) {
     var modal = $modal.open(modalOptions);
     $rootScope.$digest();
+    $timeout.flush(0);
```

Comment:
There's [a test that has a $timeout resolve](https://github.com/chrisirhc/angular-ui-bootstrap/blob/feature/fix-backdrop-animate/src/modal/test/modal.spec.js#L469-L480) , this prevents flushing that timeout so that that test is successful.

This really is supposed to just run the next tick, it's only for resolving the newly added $q promises, since there weren't any $timeout timers added.